src/NServiceBus.Core/Serializers/Json/MessageContractResolver.cs

```diff
@@ -9,7 +9,6 @@ class MessageContractResolver : DefaultContractResolver
         IMessageMapper messageMapper;
 
         public MessageContractResolver(IMessageMapper messageMapper)
-            : base(true)
```

Comment:
has that been removed with the latest version or what's the reason for removing this?